tests/Feature/FleetDispatch/FleetDispatchGenericTest.php

```diff
@@ -100,6 +104,30 @@ public function testFleetDurationCalculation(): void
         $units->addUnit(ObjectService::getShipObjectByMachineName('small_cargo'), 5);
 
         // Should take 2h:18m:05s to travel to the target planet 1 system away with base speed of 5000.
-        $this->assertEquals(8285, $fleetMissionService->calculateFleetMissionDuration($this->planetService, $targetPlanetCoords, $units));
+        $this->assertEquals(4788, $fleetMissionService->calculateFleetMissionDuration($this->planetService, $targetPlanetCoords, $units));
```

Comment:
Can you update the comment stating how long the fleet is expected to take (in hour:minute:second format)